cardano-cli/src/Cardano/CLI/Legacy/Run/Transaction.hs

```diff
@@ -123,14 +123,14 @@ runTxBuildCmd
   -> [MetadataFile]
   -> Maybe UpdateProposalFile
   -> [VoteFile In]
-  -> [NewConstitutionFile In] -- TODO: Conway era - we should replace this with a sumtype that handles all governance actions
+  -> [File () In] -- TODO: Conway era - we should replace this with a sumtype that handles all governance actions
```

Comment:
You can remove the todo

Reply:
And use the `ProposalFile In` type here as well

Reply:
I've overlooked that, thanks!